src/main/java/com/fundacionjala/movies/Customer.java

```diff
@@ -0,0 +1,69 @@
+package com.fundacionjala.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Customer {
+    private static final String BREAK_LINE = "\n";
+
+    private static final String TABULATION = "\t";
+
+    private final String name;
+
+    private List<Rental> rentals = new ArrayList<>();
+
+    public Customer(String name) {
+        this.name = name;
+    }
+
+    public void addRental(Rental rental) {
+        getRentals().add(rental);
```

Comment:
if you have the attribute of class rentals, you can use it directly instead of using the getter method.
```java
rentals.add(rental);
```